src/Compilers/Server/VBCSCompiler/NamedPipeClientConnection.cs

```diff
@@ -213,17 +152,24 @@ protected override void ValidateBuildRequest(BuildRequest request)
         /// </summary>
         private static bool ClientAndOurIdentitiesMatch(NamedPipeServerStream pipeStream)
         {
-            var serverIdentity = GetIdentity(impersonating: false);
+            if (PlatformInformation.IsWindows)
+            {
+                var serverIdentity = GetIdentity(impersonating: false);
 
-            Tuple<string, bool> clientIdentity = null;
-            pipeStream.RunAsClient(() => { clientIdentity = GetIdentity(impersonating: true); });
+                Tuple<string, bool> clientIdentity = null;
```

Comment:
We should use an actual tuple here.

Reply:
Note I didn't touch this block of code at all (the diff is just indenting it). Should I still retrofit the method with a proper tuple? (I'm not sure what our policies are for updating code as we touch surrounding areas - I generally tried to keep as hands-off as possible with this PR, otherwise it would grow really really huge with everything that needs updating/fixing)

Reply:
If you format it I think it's OK to update it as well. But I agree with the general idea of separating formatting and functional changes.